.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -21,22 +21,24 @@ Add screenshots, if applicable, to help explain your problem.
  - Device: [e.g. laptop, iPhone 8]
 
 ---
+
 [Optional] **To Reproduce**
+
 Steps to reproduce the behavior:
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
 4. See error
 
 [Optional] **Additional Context**
-Add any other context about the problem here.
+<!-- Add any other context about the problem here. -->
 
 ---
 **Contributor Resources**
-The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.
+<!-- The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.
```

Comment:
These instructions are for the contributor as the time of picking up the issue and working on it, so they will need to be able to see these as they won't know what's going on and the fact that there are two sites in this repo.

Reply:
I didn't know it was for the purpose of the contributor :+1:

Reply:
Reverted